From c81760cc3f1b6db22c7c9694efe7f3be115c2caf Mon Sep 17 00:00:00 2001
From: rofl0r <rofl0r@users.noreply.github.com>
Date: Fri, 17 May 2024 14:38:16 +0000
Subject: [PATCH] use a bigger thread stack by default

apparently newer musl versions require more stack for the TCP-based
DNS resolver.

closes #73
---
 sockssrv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/sockssrv.c
+++ b/sockssrv.c
@@ -47,7 +47,7 @@
 #endif
 
 #ifdef PTHREAD_STACK_MIN
-#define THREAD_STACK_SIZE MAX(8*1024, PTHREAD_STACK_MIN)
+#define THREAD_STACK_SIZE MAX(16*1024, PTHREAD_STACK_MIN)
 #else
 #define THREAD_STACK_SIZE 64*1024
 #endif
